### DISCUSSION TITLE
How do I contact my team members?

### DISCUSSION THREAD

**Discussion by dliseabc** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#30] [1.0 Getting started on the Data Challenge Platform - https://datascience.ey.com/]
Just wondering how can I contact my team members? I've accepted team requests from members and I've been wondering how can I contact them?

**Comment by EY-glowglowglow** [comment]
Hi there,
Just to clarify, are you looking for a collaboration space, or did you not know your team members before forming a team?

**Reply by dliseabc** [reply]
Thanks for responding. No, I don't know them before forming a team. I just clicked 'accept' from their requests. Is that not the norm?

**Reply by EY-glowglowglow** [reply]
If you send us your usernames on the platform we can get team member's emails for you so that you can contact each other. :)
eydatasciencechallenge@au.ey.com

**Comment by anprieto** [comment]
Very soon we will deploy an update so that you can contact your team members.

**Reply by dliseabc** [reply]
Sounds like a great idea. Thanks!

**Reply by anprieto** [reply]
This is already in production!!  Go talk with your team!

**Reply by ImonEmmanuel** [reply]
Has it been implemented yet because I can't talk with them.

Having a chatroom on the platform would be better

**Reply by anprieto** [reply]
Hi! Yes, a basic messaging feature has been implemented so that you can get in contact with your teammates. 
Go to Teams, there you will see this button "Send Message to team".
![image](https://user-images.githubusercontent.com/4283049/117304511-1f597200-ae7e-11eb-8654-5c0c3f7bbc1e.png)

Since this is not a full messaging system, the suggestion is that you use it just share your emails, so you can get in contact with them outside of the platform.

Creating full discussion boards and messaging is in the roadmap for next challenge.

Hope this helps.
